pyproject.toml

```
@@ -53,7 +53,7 @@ dependencies = [
 
 [project.optional-dependencies]
 default = []
-curl-cffi = ["curl-cffi==0.5.10; implementation_name=='cpython'"]
+curl-cffi = ["curl-cffi>=0.5.10,!=0.6.*,<0.8; implementation_name=='cpython'"]
```

been wanting to do this for a while, just for readability / uniformity
```suggestion
curl-cffi = [
    "curl-cffi>=0.5.10,!=0.6.*,<0.8; implementation_name=='cpython'",
]
```
of course github struggles to render it tho